Travis CI

Markel opened the pull request:
Introducing Travis CI

Markel committed:
Travis CI

Markel committed:
Updating pythons

Markel committed:
Update .travis.yml

I continue looking how can I make this work

Markel committed:
Stop emails!

Oh, and try to get 3.7 updates

Markel committed:
Solving problems (?)

MarkelFe commented:
Okay, Travis has dedicated its life to crawl metacritic twice 
[https://travis-ci.com/MarkelFe/metacritic-crawler/jobs/181965683](https://travis-ci.com/MarkelFe/metacritic-crawler/jobs/181965683)

Markel committed:
Limits and sudo

Strangly enough [python 3. 4,5,6 have failed](https://travis-ci.com/MarkelFe/metacritic-crawler/builds/102985095) in the tests, while python 3.7 and nightly so I continue trying things. Oh, and I also added some limits, because if not Travis takes 2 hours on validating a build.

Markel committed:
Change to jobs

Travis is strange

Markel committed:
.revisions

Markel committed:
Added dist xenial

Try things, it may work

Markel committed:
Removed python 3.5 builds as they don´t work properly

MarkelFe commented:
Finally, it works! Implementing Travis CI from now on!
